src/WebJobs.Script/BindingExtensions/IExtensionsManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Models;
+
+namespace Microsoft.Azure.WebJobs.Script.BindingExtensions
+{
+    public interface IExtensionsManager
+    {
+        Task<IEnumerable<ExtensionPackageReference>> GetExtensions();
```

Comment:
So where did we land on the "Async" naming convention - you through with it? :)

Reply:
I don't know, to be honest. I would like to drop it, but hate having a mix.